conftest.py

```diff
@@ -1,21 +1,39 @@
 """
-Configure the environment in which doctests run. This is necessary
-because `np` is used in doc strings for jax functions
-(e.g. `linear_transpose`) that get pulled into `scico/__init__.py`.
-
-Also allow `snp` to be used without explicitly importing, and add
-`level` parameter.
+Configure pytest.
 """
 
 import numpy as np
 
 import pytest
 
+try:
+    import ray  # noqa: F401
+except ImportError:
+    have_ray = False
+else:
+    have_ray = True
+    ray.init(num_cpus=1)  # required to be here: see ray-project/ray#44087
```

Comment:
Why `num_cpus=1`? I don't see anything about that at ray-project/ray#44087

Reply:
It's the location of the `ray.init` call that is "required to be here". The constraint on the number of cpus is so that it will play nicely with tests run in parallel.

Reply:
Updated comment in an attempt to clarify this.